TeamPage.html

```diff
@@ -200,6 +200,48 @@ <h2>Meet Our Team</h2><br>
                     </div>
                 </div>
             </div>
+            <div class="col-sm-3 col-xs" style="min-width: 210px;">
+              <div class="id_presentation">
+                <img class="img_staff" src="https://i.imgur.com/DXgnG5Y.jpg" alt="Brandon's Picture"/>
+                 <div class="text_pres">Brandon Bae<br>
+                <div class="subtext_pres">Student Programmer</div>
+                 </div>
+                 <div class="container" style="text-align: center;">
+                  <button type="button" style="margin-top: 10px;" class="btn btn-danger btn-sm" data-toggle="modal" data-target="#myModalHelenB" onclick="openBSModal('myModalJungB', this)">Brandon's Profile</button>
+                  <div class="modal fade" tabindex="-1" id="myModalHelenB" role="dialog" datim-modal>

```

Comment:
For these Bootstrap dialogs, make sure your `datim-modal` div container has a unique ID instead of "myModalHelenB". Also, the `data-target` on the button should match this new ID. Since the button's onclick already uses "myModalJungB", I recommend that!

Right now, opening Helen's profile actually opens yours because your profile appears first.